module/VuFindSearch/src/VuFindSearch/Backend/EDS/Base.php

```diff
@@ -96,6 +96,27 @@ abstract class Base implements LoggerAwareInterface
      */
     protected $searchHttpMethod = 'POST';
 
+    /**
+     * The EDS API Key for this client
+     *
+     * @var ?string
+     */
+    protected $apiKey;
+
+    /**
+     * The EDS API Key for this client (Guest Usage)
+     *
+     * @var ?string
+     */
+    protected $apiKeyGuest;
+
+    /**
+     * Indicator if user "isGuest"
+     *
+     * @var ?string
+     */
+    protected $isGuest;
```

Comment:
This has the wrong type and should also be initialized; I'd suggest defaulting to true on the principle of least privilege:

```suggestion
     * @var bool
     */
    protected $isGuest = true;
```